pykern/pkunit.py

```diff
@@ -1,10 +1,9 @@
-# -*- coding: utf-8 -*-
 """Useful operations for unit tests
 
 :copyright: Copyright (c) 2015 RadiaSoft LLC.  All Rights Reserved.
 :license: http://www.apache.org/licenses/LICENSE-2.0.html
 """
-from __future__ import absolute_import, division, print_function
+
```

Comment:
Remove

Reply:
Remove what?

Reply:
The blank space between the doc string and imports

Reply:
That's a new feature of black.